### PR TITLE
Document the current phase model

### PR DESCRIPTION
**Pull request by slomo**
In this pull request:

 - document the original modelling/constraints of phases
 - describe what has bee implemented
 - list upcoming challenges

In **future** pull requests:

 - combine plan and current state into a short term vision / documentation of desired model
 - create a long term vision from challenges, further discussion and product vision workshop

### CODE REVIEW

**Comment by xi** [comment]
I did some gardening. Sounds like a good summary to me. A bit rough on the edges, but AFAIU it will be extended later.

**Comment by slomo** [comment]
It is definitely just a first version. Nevetheless I would be interrested which topics are particular *rough on the edges*.

File: docs/phases.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+-   introduce a new dynamic way for initiators to combine phases
+    -   possibility to create custom blueprints
+    -   two step process
+    -   should support multiple modules, if implemented (see below)
+
+-   unclear concept for multiple modules
+    -   all modules need to be accessible from project detail view
+    -   modules should be integrated in the timeline
+    -   probably multiple active modules at the same time
+    -   very complex ui/ux issue
```
Comment (xi) [inline docs/phases.md]: Once the "jump to content of previous phase" issue is fixed, multiple modules yould not be an issue anymore. But "multiple phases (from different modules) at the same time" would still be one.

**Comment by xi** [comment]
> Nevetheless I would be interrested which topics are particular rough on the edges.

Thw whole text is very dense and might not be very helpful for people who are not familiar with the topic. This is especially true for the "Upcomin Challanges" section.

**Comment by slomo** [comment]
I think it is not even a text. It is intentionally just a list of bullet points. I'd prefer to keep it that way in future revisions, because that makes it easier to maintain and a more useful tool for every day work.

**Comment by 2e2a** [comment]
lgtm

**Pull request merged**